typify-impl/src/type_entry.rs

```diff
@@ -1480,17 +1486,18 @@ impl TypeEntry {
                     impl<'de> ::serde::Deserialize<'de> for #type_name {
                         fn deserialize<D>(
                             deserializer: D,
-                        ) -> Result<Self, D::Error>
+                        ) -> ::std::result::Result<Self, D::Error>
                         where
                             D: ::serde::Deserializer<'de>,
                         {
-                            String::deserialize(deserializer)?
-                                .parse()
-                                .map_err(|e: self::error::ConversionError| {
-                                    <D::Error as ::serde::de::Error>::custom(
-                                        e.to_string(),
-                                    )
-                                })
+                            Self::try_from(
```

Comment:
I'd prefer we keep this as `parse()` -- I have some ambivalence about the use of To/From<String> in these cases.

Are there other instances (I may have missed) where there's a functional change?

Reply:
No that should be the only one.  I've reverted this in a subsequent commit.